### ISSUE TITLE
Build fails from version 3.15: build-aux/install-sh: not found

### ISSUE THREAD

**Issue by Vouivre** [Limine-Bootloader/Limine#208]
If I try to build limine, if works fine with version 3.14.2 but since version 3.15 the build fails. I get the following
error:

```
./build-aux/install-sh -c -d '/home/user/.cache/kiss/proc/5926/build/limine/bin'
make[2]: Entering directory '/home/user/.cache/kiss/proc/5926/build/limine/common'
make[2]: Entering directory '/home/user/.cache/kiss/proc/5926/build/limine/common'
./build-aux/install-sh -c -d "$(dirname '/home/user/.cache/kiss/proc/5926/build/limine/decompressor-build/./entry.o')"
/bin/sh: ./build-aux/install-sh: not found
make[1]: *** [GNUmakefile:83: /home/user/.cache/kiss/proc/5926/build/limine/decompressor-build/./entry.o] Error 127
```

The full log is [here](http://pastebin.fr/108826). 

I suspect this commit introduced the problem:

https://github.com/limine-bootloader/limine/commit/4422ebd61c891613fe4f78e83d9a19e11f79b074

Unfortunately I don't understand the following:

```
MKDIR_P ?= @MKDIR_P@
export MKDIR_P
```

It could be the problem.

**Additional context**
I don't know exactly which useful information I can provide. I use the linux distribution kisslinux.

```
$ uname -a
Linux HOSTNAME 5.7.12 #7 SMP Wed Oct 21 07:21:26 UTC 2020 x86_64 GNU/Linux
```

This distribution use `musl` instead `glibc`, `busybox`replaces `coreutils`from gnu. 

If you need additionnal information, just tell me which one. 

I buid `limine`with:

```
#!/bin/sh -e

./configure \
	--prefix=/usr \
	--enable-all
	
make
make DESTDIR="$1" install
```

I use the same buildfile to build `limine`version 3.14.2 which succeeds. With the same
buildfile the the following versions which fail.

**Comment by Mintsuki** [comment]
Thanks for the report. It should be fixed in 3.16.3.

**Comment by Vouivre** [comment]
Yes, it works perfectly, thanks!
